plugins/cri/cri.go

```diff
@@ -25,13 +26,13 @@ import (
 	"github.com/containerd/plugin/registry"
 
 	containerd "github.com/containerd/containerd/v2/client"
+	srvconfig "github.com/containerd/containerd/v2/cmd/containerd/server/config"
```

Comment:
this felt a tad wrong ... importing from `cmd/containerd` into here. but we have done this elsewhere so it's ok.

Reply:
We can probably safely move that config package out from there. It got moved as a subpackage but considering it is our main configuration object, it could find a better home.

Reply:
I'll move the configuration version out into the version package in a follow up. That's only thing I saw being imported from config

Reply:
thanks!